Library/Formula/qt-gstreamer.rb

```diff
@@ -0,0 +1,66 @@
+class QtGstreamer < Formula
+  desc "C++ bindings for GStreamer with a Qt-style API"
+  homepage "http://gstreamer.freedesktop.org/modules/qt-gstreamer.html"
+  url "http://gstreamer.freedesktop.org/src/qt-gstreamer/qt-gstreamer-1.2.0.tar.xz"
+  sha256 "9f3b492b74cad9be918e4c4db96df48dab9c012f2ae5667f438b64a4d92e8fd4"
+  head "git://anongit.freedesktop.org/gstreamer/qt-gstreamer"
+
+  option "with-static", "Build QtGStreamer as a static library"
+  option "with-tests", "Build QtGStreamer's tests"
+  option "without-examples", "Build QtGstreamer's examples"
+  option "with-codegen", "Build and use QtGStreamer's codegen"
+  option "with-qt5", "Build against Qt5"
+  option "with-qt-dir", "Add supply CMake with qmake of the Qt installation to"
+         "use. Specify it in the environment variable HOMEBREW_QTGSTREAMER_QT."
+         "Use this options to build against Qt versions not installed with"
+         "homebrew."
```

Comment:
Oh, ok. It's just that `brew info qt5` says "Qt 5 conflicts Qt 4". I was about to file a bug because qt does not conflict with qt5 and installs after qt5 has been installed.